samples/samples/autocommit.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+# [START enable_autocommit_mode]
+def enable_autocommit_mode(instance_id, database_id):
+    """ Enables autocommit mode. """
```

Comment:
Excess whitespaces at the start and the end of the docstring

Reply:
Done